posts/using-napari-for-cosmx-data/index.qmd

```diff
@@ -0,0 +1,525 @@
+```
+
+:::
+
+#### Part 2: Installing `napari-cosmx`
+
+1. Download the whl file (**need a location for this**). 
+2. Go to the folder where the `whl` file is in your File Explorer and choose 
+`Copy Path`.
+2. Now launch napari from the Start menu in Windows. 
+
+> Pro tip: If launching for the first time, the application may take a moment to appear. Avoid launching multiple instances. 
+
+3. In napari, open up the `>_` button that is located on the bottom left (see @fig-windows-plugin-command for example). 
+4. Type `pip install ` into the console (_i.e._, with a single space after the word 'install').
+5. Paste the location of the `whl` file
+6. Press enter to execute
+7. You should receive a message in the console that several packages were successfully
+installed including the `napari-cosmx` package. 
+8. **Close and re-start napari**
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-windows-plugin-command"
+#| fig-cap: "Example showing how to install the `napari-cosmx` file. Your file name and path will look different."
+
+knitr::include_graphics("./figures/windows_plugin_command.png")
+```
+
+
+::::
+
+
+
+
+### MacOS/Unix Install
+
+::: {.callout-note}
+Depending on your browser and security settings you may get warnings when downloading or running some of the links below.
+:::
+
+#### Part 1: Installing Napari
+
+Visit the napari [0.4.17 release page](https://github.com/napari/napari/releases/tag/v0.4.17){target="_blank"}. 
+Scroll all the way to the bottom and expand the section that says "Assets" (@fig-mac-pkg-download). 
+Download the `pkg` file that is appropriate for your operating system (_i.e._, 
+`napari-0.4.17-macOS-x86_64.pkg` for Mac). Open the downloaded file and install 
+via the instructions on screen (you can accept the defaults). When finished, 
+launch napari via the Applications folder. 
+
+> Pro tip: If launching for the first time, the application may take a moment to appear. Avoid launching multiple instances. 
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-mac-pkg-download"
+#| fig-cap: "Screenshot of napari packages. Blue highlighted package is appropriate for Mac. Other operating systems' packages are also available but untested."
+
+knitr::include_graphics("./figures/mac_install_pkg.png")
+```
+
+
+
+#### Part 2: Installing `napari-cosmx`
+
+1. In napari, open up the `>_` button that is located on the bottom left (see @fig-mac-plugin-command for example). 
+2. Type `pip install ` into the console (_i.e._, with a single space after the word 'install').
+3. Paste the location of the `whl` file
+4. Press enter to execute
+5. You should receive a message in the console that several packages were successfully
+installed including the `napari-cosmx` package. 
+6. **Close and re-start napari**
+
+```{r}
+#| eval: true
+#| echo: false
+#| label: "fig-mac-plugin-command"
+#| fig-cap: "Example showing how to install the `napari-cosmx` file. Your file name and path will look different."
```

Comment:
Can the text in figure be made any bigger? Hard to read.
Also add to legend: 'Console open button is circled in yellow and path is underlined.'

Reply:
I'm not sure the yellow underline is needed. The one from the Windows image was a legacy from an earlier screenshot. I'll can remove it from the Windows image. I have updated with a slightly larger image but I'm limited in what elements and panels napari lets displayed.

Reply:
1553b522513e0a2fa52178ad4b77fe786585220a

Reply:
2c07f5b40f6bc1f27aaa6391ed017a61ec807857